README.md

```diff
@@ -36,13 +36,16 @@ Cut the connector off one one end of the RJ11/12 cable, then strip and connect t
                +---------+
             1  ---       |
             2  ---       +--+      
-Red     B   3  ---          |     
-Green   A   4  ---          |        
-Yellow  GND 5  ---       +--+
+        B   3  ---          |     
+        A   4  ---          |        
+       GND 5  ---       +--+
```

Comment:
Add space to fix alignment

Reply:
I was so focused on the pic I botched the ascii art!